Jenkinsfile

```diff
@@ -81,28 +72,26 @@ podTemplate(cloud: 'openshift', label: 'coreos-assembler', yaml: pod, defaultCon
           }
         }
 
-        // Special case for devel pipelines not running in our project and not
-        // uploading to S3; in that case, the only way to make the builds
-        // accessible at all is to have them in the PVC.
-        if (!prod && !prod_jenkins && !s3_builddir) {
-            utils.workdir = "/srv"
-        }
-
         stage('Init') {
 
             def ref = params.STREAM
             if (src_config_ref != "") {
+                assert !prod : "Asked to override ref in prod mode"
                 ref = src_config_ref
             }
 
-            utils.shwrap("""
-            # just always restart from scratch in case it's a devel pipeline
-            # and it changed source url or ref; this info also makes it into
-            # the build metadata through cosa reading the origin remote
-            rm -rf src/config
+            // for now, just use the PVC to keep cache.qcow2 in a stream-specific dir
+            def cache_img
+            if (prod) {
+                cache_img = "/srv/prod/${params.STREAM}/cache.qcow2"
+            } else {
+                cache_img = "/srv/devel/${devel_prefix}/cache.qcow2"
```

Comment:
wondering if we can make this `/srv/devel/${devel_prefix}/${params.STREAM}/cache/cache.qcow2`

and then below store the builds in `/srv/devel/${devel_prefix}/${params.STREAM}/builds/`

Reply:
Hmm, though in devel mode, the "stream" doesn't mean anything since you provide the ref to check out. See also https://github.com/coreos/fedora-coreos-pipeline/blob/e50e6780972c3c29b6c3fb529136fc81972da174/Jenkinsfile#L75. Does that make sense?

Reply:
yeah - it was just a gut reaction when I started digging in the PVC. seemed like the stream (ref) should be in the path